src/main/java/uk/gov/pay/api/model/CreateDirectDebitPaymentRequest.java

```diff
@@ -45,21 +47,25 @@ public CreateDirectDebitPaymentRequest() {
     }
     
     @ApiModelProperty(value = "amount in pence", required = true, allowableValues = "range[1, 10000000]", example = "12000")
+    @Schema(description = "amount in pence", required = true, example = "12000")
```

Comment:
should there be `minimum` and `maximum` properties on this to replace the `allowableValues`?

Reply:
Good catch , updated